src/utils/validateConfigOverride.js

```diff
@@ -0,0 +1,15 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { anything, mapOfValues } from "./validation";
+
+export default mapOfValues(mapOfValues(anything()));
```

Comment:
Are we sure there will never be a top level value that isn't an object?

Reply:
There are a few non-object top leve-enums ([schema](https://git.corp.adobe.com/experience-edge/blackbird/blob/master/manifests/src/main/resources/schemas/apiv2/edge/datastreams/v1/edgeSettings.json)) that could be potentially supported in the future:

- `type`
- `access_type`
- `geo_resolution`
- `default_consent`

as well as a few properties that are objects that don't start with `com_adobe_`:

- `annotations`
- `state`
- `input`

Any suggestions on how to write the validation (and `prepareConfigOverridesForEdge()`) without coupling the web sdk too much to the Blackbird schema structure?